Software/Human-Zombie-Device/ReceivedPacket.cpp

```diff
@@ -0,0 +1,14 @@
+#include "ReceivedPacket.h"
+
+int ReceivedPacket::getRssi() {
+    return this->rssi;
+}
+String ReceivedPacket::getDeviceName() {
+    return this->deviceName;
+}
+void ReceivedPacket::setRssi(int rssi) {
+    this->rssi = rssi;
+}
+void ReceivedPacket::setDeviceName(String deviceName) {
```

Comment:
You may want to add an if check for if (deviceName == "") - Set Default.

Reply:
added this without adding a dependency to DeviceState.h because I don't want to litter the code with that include. However, it might not be the best way because no we have a "magic string". I was in two minds to just leave it as blank entirely but I didn't. Thatll do for now, can always get cleaned up later if we do better error handling in devicestate.